js-weather.js

```diff
@@ -0,0 +1,50 @@
+const escapeHtml = (str) => {
+  return str.toString().replace(/^\s+|\s+$/g,'');
+}
+
+const showTodaysWeather = (response) => {
+  const obj = JSON.parse(response);
+
+  const city = obj.name;
+  document.getElementById('selectedCity').textContent = `現在の${escapeHtml(city)}の天気は...`;
+
+  const weather = obj.weather[0].description;
+  document.getElementById('weather').innerHTML = `<span>${escapeHtml(weather)}</span>`;
+
+  const temperature = obj.main.temp;
+  document.getElementById('temperature').innerHTML = `<span>${escapeHtml(temperature)}℃</span>`;
+
+  const weatherIcon = obj.weather[0].icon;
+  document.getElementById('weather-icon').innerHTML = `<img src='http://openweathermap.org/img/w/${escapeHtml(weatherIcon)}.png' >`;
+}
```

Comment:
```suggestion
};
```
これも流派があるのですが、このファイルでは他の行に合わせて文末にセミコロンを付けた方がよいでしょう